lib/ui/map/map_view.dart

```diff
@@ -57,6 +61,10 @@ class MapViewState extends State<MapView> {
       Provider.of<UserRepository>(context, listen: false)
           .setUserPosition(userPosition);
     });
+
+    WidgetsBinding.instance.addPostFrameCallback((_) => _postFrameCallBack());
+
+    super.initState();
```

Comment:
move initState up since according to flutter.dev: "If you override this, make sure your method starts with a call to super.initState()."

https://api.flutter.dev/flutter/widgets/State/initState.html

Reply:
Tried that, but it resulted in the callback not being called :(